app-admin/pwman3/pwman3-0.10.0-r1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+PYTHON_REQ_USE="sqlite?"
```

Comment:
And if you remove the sqlite flag below, this won't work. Is this needed by pwman3 or whatever sqlite used to pull?

Reply:
You need sqlite to run pwman3 - unless you use another database.